sale_channel_partner_binding/models/sale_order.py

```diff
@@ -0,0 +1,10 @@
+# Copyright 2020 Akretion
```

Comment:
Not needed

Reply:
I think you mis-undertand, I mean the do not need this file. adding partner_binding_id is useless.
But you should keep the copyright header  of Akretion ;)